backend/legacyapi/setting.go

```diff
@@ -33,6 +35,13 @@ type IMType string
 // IMTypeFeishu is IM feishu.
 const IMTypeFeishu IMType = "im.feishu"
 
+// WorkspaceGeneralSettingPayload is the payload for SettingWorkspaceGeneral.
+type WorkspaceGeneralSettingPayload struct {
```

Comment:
Let's use proto to represent the json to be stored in payload.

https://sourcegraph.com/github.com/bytebase/bytebase/-/tree/proto/store

Reply:
Done